### PR TITLE
Use JS modules

### PR DESCRIPTION
**Pull request by Kwaleyela-Ikafa**
## in this branch we:
- Used ES6 modules

### CODE REVIEW

File: modules/addBook.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  e.preventDefault();
+}
+
+function removeBook(e) {
+  if (e.target.className === 'remove remove-btn btn') {
+    ui.removeBookFromUI(e.target);
+    Storage.removeFromBooks(e.target);
+  }
+}
```
Comment (omarsalem7) [inline modules/addBook.js]: - [x] Kindly use arrow syntax for addBook and removeBook like this: `const addBook = (e) => { }` as stated in the requirements.

image

File: modules/navigation.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-unused-vars */
+import {
+  BookData, button, titleInput, authorInput, Storage,
+} from './storage.js';
+import { Book, UI, ui } from './UI.js';
+import { addBook, removeBook } from './addBook.js';
```
Comment (omarsalem7) [inline modules/navigation.js]: - [x] Kindly remove these unused codes from your code to make it clean.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Kwaleyela-Ikafa 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- The current date is displayed below the navigation bar ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @Kwaleyela-Ikafa,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ ES6 syntax

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**